src/ui/components/Library/Team/View/TestRuns/TestRunsPage.tsx

```diff
@@ -80,9 +80,9 @@ function TestRunsContent() {
 
         {focusId && (
           <>
-            <PanelResizeHandle className="h-full w-4" />
+            <PanelResizeHandle className="w-2 h-full" />
```

Comment:
I think this is the only thing that changed in the whole file: "w-4" -> "w-2"

### Before
```js
<PanelResizeHandle className="h-full w-4" />
```

### After
```js
<PanelResizeHandle className="h-full w-2" />
```

Reply:
Yup!